TLM/TLM/State/OptionsTabs/MaintenanceTab_ConfigGroup.cs

```diff
@@ -41,6 +47,10 @@ private static void OnReloadGlobalConfigClicked()
             => GlobalConfig.Reload();
 
         private static void OnResetGlobalConfigClicked()
-            => GlobalConfig.Reset(oldConfig: null, resetAll: true);
+            => GlobalConfig.Reset(oldConfig: null);
+        private static void OnSetAsDefaultForNewGamesClicked() {
+            GlobalConfig.Instance.SavedGameOptions = SavedGameOptions.Instance;
```

Comment:
Why the property is named like that? Isn't it better to have something like: `DefaultGameOptions` instead?